admin.php

```diff
@@ -46,9 +46,22 @@
 <script type="text/javascript" src="js/plugins/jquery.js"></script>
 <script type="text/javascript" src="js/plugins/thickbox.js"></script>
 <script type="text/javascript" src="js/common.js"></script>
+<script type="text/javascript" src="js/plugins/Gettext.js"></script>
+<?php
+   if(isset($_COOKIE["lang"])){
+        if($_COOKIE["lang"]=='fr'){

```

Comment:
Yes @remocrevo , I work in this patch and I find a solution, I will test this and add the update of database to change 'latin1' to 'utf8' for all modules in another patch, thanks